ultralytics/solutions/serve.py

```diff
@@ -0,0 +1,29 @@
+import base64
+import io
+
+import litserve as ls
+import numpy as np
+from PIL import Image
+
+from ultralytics import YOLO
+
+
```

Comment:
Would be great if request would include the task 💯 

```suggestion
from pydantic import BaseModel

class UltralyticsRequest(BaseModel):
    images: List[str]
    task: str = "detection"

```